TypeCobol/Compiler/SqlAnalysis/SqlQualityAnalyzer.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+using TypeCobol.Analysis;
+using TypeCobol.Compiler.Parser;
+
+namespace TypeCobol.Compiler.SqlAnalysis
+{
+    /// <summary>
+    /// SQL Quality Analyzer class.
+    /// </summary>
+    public class SqlQualityAnalyzer : QualityAnalyzerBase
```

Comment:
Not needed, now that the SQL code is regular part of the AST, it will be inspected during the visit of the ProgramClassDocument.

Reply:
In fact this act as the dedicated analyzer class that is registered in CLI like:
analyzerProviderWrapper.AddActivator(options => new SqlQualityAnalyzer());
For for this POC is only to show that concepts work.

Reply:
Yes. As we have checked that it works, you can now remove everything related to sql analysis from the branch.

Reply:
done in [7f2afb7](https://github.com/TypeCobolTeam/TypeCobol/pull/1928/commits/7f2afb778747d8f26ff0ce751781c2ab8845aa78)